com.unity.render-pipelines.universal/Runtime/ForwardRenderer.cs

```diff
@@ -135,6 +136,10 @@ protected override void Dispose(bool disposing)
         /// <inheritdoc />
         public override void Setup(ScriptableRenderContext context, ref RenderingData renderingData)
         {
+#if UNITY_2020_2_OR_NEWER
```

Comment:
TBH I am not fan of spaming our code base with this defines, makes code harder to read :/

Reply:
The other option is to indent all of the code (wrap in {} blocks) :(
If we know these aren't backported we can remove those ifs.

Reply:
Done. removed the #ifs